brave/brave-ios
# Fix #4438: Downloads Improvements

## Summary of Changes


This pull request fixes #4438

## Submitter Checklist:


## Test Plan:

Test 1:
- Fresh Install App
- Go to settings menu and press Downloads
- Check a Downloads folder will be created inside Brave folder for File.app 

Test2: 
- Download a file
- Check the download progress and press downloads in that toast
- Press Downloads
- Check File.app is opened pointing to Dowloads folder inside Brave folder
- Check downloaded file exists in the folder.

## Screenshots:

https://user-images.githubusercontent.com/6643505/147136388-4ff9d765-87a4-470d-ad1c-23b7197acca2.MP4




## Reviewer Checklist:

  - `QA/(Yes|No)`
  - `bug` / `enhancement`
